api/src/main/java/io/strimzi/api/kafka/model/KafkaResources.java

```diff
@@ -0,0 +1,187 @@
+     * @param podNum The number of the Kafka pod
+     * @return The name of the corresponding Kafka {@code Pod}.
+     */
+    public static String kafkaPodName(String clusterName, int podNum) {
+        return kafkaStatefulSetName(clusterName) + "-" + podNum;
+    }
+
+    /**
+     * Returns the name of the Entity Operator {@code Deployment} for a {@code Kafka} cluster of the given name.
+     * This {@code Deployment} will only exist if {@code Kafka.spec.entityOperator} is configured in the
+     * {@code Kafka} resource with the given name.
+     * @param clusterName  The {@code metadata.name} of the {@code Kafka} resource.
+     * @return The name of the corresponding Entity Operator {@code Deployment}.
+     */
+    public static String entityOperatorDeploymentName(String clusterName) {
+        return clusterName + "-entity-operator";
+    }
+
+    /**
+     * Returns the name of the Cluster CA certificate {@code Secret} for a {@code Kafka} cluster of the given name.
+     * @param clusterName  The {@code metadata.name} of the {@code Kafka} resource.
+     * @return The name of the corresponding Cluster CA certificate {@code Secret}.
+     */
+    public static String clusterCaCertificateSecretName(String clusterName) {
+        return clusterName + "-cluster-ca-cert";
+    }
+
+    /**
+     * Returns the name of the Cluster CA key {@code Secret} for a {@code Kafka} cluster of the given name.
+     * @param clusterName  The {@code metadata.name} of the {@code Kafka} resource.
+     * @return The name of the corresponding Cluster CA key {@code Secret}.
+     */
+    public static String clusterCaKeySecretName(String clusterName) {
+        return clusterName + "-cluster-ca";
+    }
+
+    /**
+     * Returns the name of the Clients CA certificate {@code Secret} for a {@code Kafka} cluster of the given name.
+     * @param clusterName  The {@code metadata.name} of the {@code Kafka} resource.
+     * @return The name of the corresponding Clients CA certificate {@code Secret}.
+     */
+    public static String clientsCaCertificateSecretName(String clusterName) {
+        return clusterName + "-clients-ca-cert";
+    }
+
+    /**
+     * Returns the name of the Clients CA key {@code Secret} for a {@code Kafka} cluster of the given name.
+     * @param clusterName  The {@code metadata.name} of the {@code Kafka} resource.
+     * @return The name of the corresponding Clients CA key {@code Secret}.
+     */
+    public static String clientsCaKeySecretName(String clusterName) {
+        return clusterName + "-clients-ca";
+    }
+
+    /**
+     * Returns the name of the internal bootstrap {@code Service} for a {@code Kafka} cluster of the given name.
+     * @param clusterName  The {@code metadata.name} of the {@code Kafka} resource.
+     * @return The name of the corresponding bootstrap {@code Service}.
+     */
+    public static String internalBootstrapServiceName(String clusterName) {
+        return clusterName + "-kafka-bootstrap";
```

Comment:
I think the name `internal` suggests it should not be used ... do you think it will be clear?

Reply:
Yeah, that's a possible misinterpretation. I only added internal for symmetry with external. I'm happy to change it.